source/_integrations/ping.markdown

```diff
@@ -47,6 +47,11 @@ name:
   required: false
   type: string
   default: Ping [hostname]
+scan_interval:
```

Comment:
Hey there @fschild 👋 

I get that you want to add it here, but actually, `scan_interval` is not a option of `ping` at all.

You can find the documentation for `scan_interval` here: <https://www.home-assistant.io/docs/configuration/platform_options/#scan-interval>

It is part of the platform configuration.

Nevertheless, thanks for willing to contribute 👍

Reply:
This may be the technical correct configuration/answer because this option exists due to inheritance from a different component.

BUT, as a user I do not care. I just want a list of all options I can use in *.yaml with the component.

Reply:
> this option exists due to inheritance from a different component

Partially, it's not a different component, is a base configuration of Home Assistant and applies to all.

> BUT, as a user I do not care.

I can understand that reasoning. The good news is though, this is one of the integrations that will move away from YAML (as it connects to/polls a device).

That will remove part of the standards issues, you are now complaining about :)

Reply:
> GUI will allow to set the scan-interval.

Almost all integration can be manually polled using simple automations that trigger on a time interval (or anything you like as you define the trigger in an automation), and call the `homeassistant.update_entity` service on one of the entities of the integration you'd like to trigger a poll on.

There are no options needed for that.